src/components/candidateCard/candidateInfo/CandidateInfo.jsx

```diff
@@ -26,18 +26,20 @@ export const CandidateInfo = () => {
         <Typography paddingLeft="1%" variant="h4">
           {`${candidate.firstName} ${candidate.lastName}`}
         </Typography>
```

Comment:
think about how you can refactor this file

Reply:
This bug will fix @oleksiuk-ol